### PR TITLE
update vraptor version compatible

### PR DESCRIPTION
**Pull request by felipeweb**

### CODE REVIEW

File: src/main/java/br/com/caelum/vraptor/restfulie/serialization/XStreamBuilderImpl.java [hunk 1]
```diff
@@ -20,6 +22,8 @@
 
 	private final XStreamConverters converters;
 	private final TypeNameExtractor extractor;
+	private final Serializee serializee;
+	private final ReflectionProvider reflectionProvider;
```
Comment (Turini) [inline src/main/java/br/com/caelum/vraptor/restfulie/serialization/XStreamBuilderImpl.java]: unused?

Comment (felipeweb) [inline src/main/java/br/com/caelum/vraptor/restfulie/serialization/XStreamBuilderImpl.java]: sorry i`ll remove unused parameter

2015-01-21 14:53 GMT-02:00 Rodrigo Turini <notifications@github.com>:

> In
> src/main/java/br/com/caelum/vraptor/restfulie/serialization/XStreamBuilderImpl.java
> <https://github.com/caelum/vraptor-restfulie/pull/6#discussion_r23314047>:
>
> > @@ -20,6 +22,8 @@
> >
> >  	private final XStreamConverters converters;
> >  	private final TypeNameExtractor extractor;
> > +	private final Serializee serializee;
> > +	private final ReflectionProvider reflectionProvider;
>
> unused?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caelum/vraptor-restfulie/pull/6/files#r23314047>.
>



-- 
Felipe Oliveira
Tel: 11 99930-9974
Caelum | Ensino & Inovação
www.caelum.com.br

Comment (felipeweb) [inline src/main/java/br/com/caelum/vraptor/restfulie/serialization/XStreamBuilderImpl.java]: done!

2015-01-21 15:00 GMT-02:00 Felipe Oliveira <felipeweb.programador@gmail.com>
:

> sorry i`ll remove unused parameter
>
> 2015-01-21 14:53 GMT-02:00 Rodrigo Turini <notifications@github.com>:
>
> In
>> src/main/java/br/com/caelum/vraptor/restfulie/serialization/XStreamBuilderImpl.java
>> <https://github.com/caelum/vraptor-restfulie/pull/6#discussion_r23314047>
>> :
>>
>> > @@ -20,6 +22,8 @@
>> >
>> >  	private final XStreamConverters converters;
>> >  	private final TypeNameExtractor extractor;
>> > +	private final Serializee serializee;
>> > +	private final ReflectionProvider reflectionProvider;
>>
>> unused?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/caelum/vraptor-restfulie/pull/6/files#r23314047>.
>>
>
>
>
> --
> Felipe Oliveira
> Tel: 11 99930-9974
> Caelum | Ensino & Inovação
> www.caelum.com.br
>



-- 
Felipe Oliveira
Tel: 11 99930-9974
Caelum | Ensino & Inovação
www.caelum.com.br

**Comment by Turini** [comment]
great, thanks!

**Pull request merged**